docs/_docs/versions.md

```diff
@@ -5,114 +5,157 @@ layout: post-toc
 redirect_from: /docs/
 ---
 
-# Manage Versions of Your Zapier Integration
+# How to update your integration on the Zapier Developer Platform
```

Comment:
I don't think we should change the title here - this doc really is about managing versions, including things like deprecation that, while related to updates, aren't intrinsically connected.

Reply:
I agree on keeping Manage Versions but like the inclusion of Edit/Update verbiage for readers unclear on how "versions" affect them - what about "Manage Edits/Updates to your Integration with Versions"?